docs/BasicCommunication/OnSystemTimeReady/index.md

```diff
@@ -0,0 +1,37 @@
+## OnSystemTimeReady
+
+Type
+: Notification
+
+Sender
+: HMI
+
+Purpose
+: Inform SDL about readiness to provide system time
```

Comment:
Inform SDL about readiness to provide system time
to
Notify SDL that the HMI is ready to provide accurate system time.

Reply:
@mrapitis, thank you.
fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/94/commits/671b954b558288f41dc50414381000896037d493